fooof/objs/fit.py

```diff
@@ -1098,8 +1103,14 @@ def _drop_peak_overlap(self, guess):
     def _calc_r_squared(self):
         """Calculate the r-squared goodness of fit of the model, compared to the original data."""
 
+        # compute r-squared
         r_val = np.corrcoef(self.power_spectrum, self.fooofed_spectrum_)
         self.r_squared_ = r_val[0][1] ** 2
+        
+        # compute adjusted r-squared
+        n = len(self.power_spectrum) # number of data points
+        k = len(self.peak_params_) * 3 + 2 # number of parameters
```

Comment:
@mwprestonjr - just checking in here: the number of parameters here is computed as n_peaks * 3 (reflecting the 3 Gaussian params per Gaussian), and then + 2, presumably for aperiodic - but I think this should be +2 if fixed, +3 if knee mode, right? Or, more generally, `n_params = n_peaks * n_params_per_peak + n_ap_param`?

Reply:
Hey Tom, nice catch. Sorry I missed this. I've now made this update.